### PR TITLE
Update stepup config after install

### PR DESCRIPTION
**Pull request by pablothedude**
Make some changes in the Setpup config after first install

- Update documentation
- Add MariaDB healthcheck  to prevent running commands when not healthy
- Cleanup duplicate Behat docker compose config

### CODE REVIEW

File: stepup/README.md [hunk 1]
```diff
@@ -117,16 +117,17 @@ STEPUP_VERSION=test
 3. Choose if you want to run the containers in the back- or foreground.
 
 3: Once the containers are up and running, you can run the behat tests
-1. Open a shell in the `behat` container `$ docker exec -it stepup-behat-1 bash`
-2. Run the tests:
+1. Install the required dependencies in the container `$ docker compose exec composer install`
```
Comment (quartje) [inline stepup/README.md:120]: docker compose exec behat composer install

**Review by quartje — APPROVED** [review]
Nice work! 
The extra behat composer file was there so that container would only be started when running tests and not in other circumstances. But I guess we can better solve that using docker compose profiles

**Comment by pablothedude** [comment]
> Nice work! The extra behat composer file was there so that container would only be started when running tests and not in other circumstances. But I guess we can better solve that using docker compose profiles

I saw that the config was already present in the docker compose file and the separate file was a duplicate because the contents were the same.

File: stepup/README.md (hunk 1 above)
Comment (pablothedude) [reply stepup/README.md]: Thanks, changed the command

**Review by MKodde — APPROVED** [review]
No objections your honor

**Pull request merged**